content/services/services_ibm_z.rst

```diff
@@ -0,0 +1,29 @@
+IBM Z Development Hosting
+============================
+:slug: services/ibm-z
+:title: IBM Z Development Hosting
+:menu: Services, IBM Z Development Hosting, 4
+
+The Open Source Lab partners with `IBM`_ to host `IBM Z`_ based servers in order
+to provide an open platform for innovation to the open source community. With
+this hardware, the Open Source Lab offers continuous integration services via
+Jenkins allowing open source projects to build and test on the s390x architecture.
+
+.. _IBM: http://www-03.ibm.com/linux/ltc/
+.. _IBM Z: https://en.wikipedia.org/wiki/IBM_Z
+
+IBM Z Continuous Integration (IBM Z CI)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+An OSL-managed Jenkins service is hosted at https://powerci.osuosl.org. This
```

Comment:
We're actually going to be deploying a new CI instance just for the Z jobs. We'll likely go with https://ibmz-ci.osuosl.org